### PR TITLE
Add section for 2024-5.1

### PR DESCRIPTION
**Pull request by heratyian**

### CODE REVIEW

**Review by aizatibraimova — COMMENTED** [review]
looks good

**Review by SaraDawner2000 — APPROVED** [review]
Looks great!

**Review by APatel-AI — APPROVED** [review]
All Good!!

**Review by jniewiel — APPROVED** [review]
Looks fantastic!

**Review by aizatibraimova — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -96,4 +96,9 @@ Clone this repository (or open in codespace), create a branch with your name, ad
 | Benjamin Guzman      | I have met Gherbo before|
 | Alana Edwards        | My favorite color is pink. |
 | Crystal Tliourane    | I enjoy spicy foods! |
-| Lamont Sampson       | Earth is pretty big, but it's tiny compared to the rest of the universe.  For perspective : Around 1,000 Earths can fit inside Jupiter.  Around 1,000 Jupiters can fit inside the sun and over 1 billion suns can fit inside the star Betelgeuse. |
\ No newline at end of file
+| Lamont Sampson       | Earth is pretty big, but it's tiny compared to the rest of the universe.  For perspective : Around 1,000 Earths can fit inside Jupiter.  Around 1,000 Jupiters can fit inside the sun and over 1 billion suns can fit inside the star Betelgeuse. |
+
+## 2024-5.1
+
+| Name                 | Fun Fact |
```
Comment (jniewiel) [inline README.md:103]: Looks good.

**Pull request merged**